website/project/forms.py

```diff
@@ -16,4 +17,5 @@ class NewNodeForm(Form):
         validators.Length(max=200, message=u'Title must contain fewer than 200 characters.')
     ], widget=BootstrapTextInput())
     description = TextAreaField('Description', widget=BootstrapTextArea())
-    category = TextAreaField('Category', widget=BootstrapTextArea())
+    category = TextField('Category', widget=BootstrapTextInput())
+    inheritContributors = TextField('Inherit', widget=BootstrapTextInput())
```

Comment:
I really don't understand the relationship between the `TextField` and `BootstrapTextInput` classes here, and the HTML that is showing up in the browser: a dropdown and a checkbox!

![screen shot 2016-02-10 at 10 45 12 am](https://cloud.githubusercontent.com/assets/134455/12952132/6a94028c-cfe3-11e5-867f-d12cabbbb7ca.png)